src/lib/turtle/__init__.js

```diff
@@ -1017,28 +1017,6 @@ function generateTurtleModule(_target) {
         };
         proto.$shape.minArgs = 0;
 
-        proto.$window_width = function() {
-            return this._screen.$window_width();
-        };
-
-        proto.$window_height = function() {
-            return this._screen.$window_height();
-        };
-
-        proto.$tracer = function(n, delay) {
```

Comment:
As an experiement, I just added all of these back into the source at the spot where they were removed.  All appears well.  Although there may be a different problem that I overlooked this is certainly an easy solution.

Brad